### ISSUE TITLE
[NBug] Index was outside the bounds of the array.

### ISSUE THREAD

**Issue by pashokred** [gitextensions/gitextensions#10493]
### Environment

- Git Extensions 4.0.0.15569
- Build cba315df5366fca01240d93b6e5a5077fd5e6630
- Git 2.38.1.windows.1
- Microsoft Windows NT 10.0.19045.0
- .NET 6.0.11
- DPI 96dpi (no scaling)

### Issue description

```
System.IndexOutOfRangeException: Index was outside the bounds of the array.
   at GitExtensions.Plugins.BackgroundFetch.BackgroundFetchPlugin.<>c__DisplayClass11_0.<RecreateObservable>b__1(Int64 i)
   at System.Reactive.AnonymousSafeObserver`1.OnNext(T value) in /_/Rx.NET/Source/src/System.Reactive/AnonymousSafeObserver.cs:line 54
   at System.Reactive.ObserveOnObserverLongRunning`1.Drain() in /_/Rx.NET/Source/src/System.Reactive/Internal/ScheduledObserver.cs:line 732
   at System.Reactive.ObserveOnObserverLongRunning`1.<>c.<.cctor>b__17_0(ObserveOnObserverLongRunning`1 self, ICancelable cancelable) in /_/Rx.NET/Source/src/System.Reactive/Internal/ScheduledObserver.cs:line 680
   at System.Reactive.Concurrency.NewThreadScheduler.<>c__DisplayClass7_0`1.<ScheduleLongRunning>b__0() in /_/Rx.NET/Source/src/System.Reactive/Concurrency/NewThreadScheduler.cs:line 94
   at System.Threading.Thread.StartHelper.Callback(Object state)
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
--- End of stack trace from previous location ---
   at System.Threading.Thread.StartCallback()
```

Set periodic background fetch to every 20 seconds, after that set settings from effective to global for all repositories and click Apply

### Steps to reproduce

Set periodic background fetch to every 20 seconds, after that set settings from effective to global for all repositories and click Apply

**Comment by mstv** [comment]
Duplicate of #10323
Fixed by #10445

**Comment by mstv** [comment]
I am sure, *I* have clicked "Close with comment", @github.
